config/webpack.config.js

```diff
@@ -27,6 +27,18 @@ const configGenerator = (options) => {
   if (options.entry) {
     filesToBuild = _.pick(entryFiles, options.entry.split(',').map(x => x.trim()));
   }
+  filesToBuild.vendor = [
+    'core-js',
+    'history',
+    'lodash',
```

Comment:
We might want to skip lodash here. We currently have babel and webpack plugins that transform it so we're only including the specific functions in use, but this will include all of lodash for every app.